src/doc/book/src/procedural-macros.md

```diff
@@ -210,4 +210,74 @@ Hello, World! My name is FrenchToast
 Hello, World! My name is Waffles
 ```
 
-We've done it!
```

Comment:
Removing this for the next section seems to make the previous section truncated, IMO removal is not needed.